concurrency-limiter/concurrency_limiter_test.go

```diff
@@ -11,6 +11,59 @@ import (
 	"time"
 )
 
+func Test_Met_True_When_OverLimit(t *testing.T) {
```

Comment:
You should add a test  case for the default 0 value, to make the default behavior clear and prevent any breaking changes.